### PR TITLE
categorize some qDebugs

### PR DESCRIPTION
**Pull request by sethalves**
- categorized some qDebugs so that QT_LOGGING_RULES environment variable can be used to cut down on interface log spam

### CODE REVIEW

**Comment by sethalves** [comment]
to test:
- no functional change

**Comment by hifi-gustavo** [comment]
Build console output for commit: 44703386634f112b3ae8647f197bae151c6d42b1 available at: https://gist.github.com/18caf6aac16455780aab75e4afd7f406<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8603-4470338.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8603-4470338.dmg

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -264,7 +264,7 @@ class DeadlockWatchdogThread : public QThread {
             auto elapsedMovingAverage = _movingAverage.getAverage();
 
             if (elapsedMovingAverage > _maxElapsedAverage) {
-                qDebug() << "DEADLOCK WATCHDOG WARNING:"
+                qCDebug(interfaceapp) << "DEADLOCK WATCHDOG WARNING:"
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: I would recommend that we use a more narrow category for this... maybe "deadlock"?

Comment (sethalves) [inline interface/src/Application.cpp]: okay, those are now in interfaceapp_deadlock

**Comment by hifi-gustavo** [comment]
Build console output for commit: d67701f7bb963d4aad5344c568c21d393e8fe3b1 available at: https://gist.github.com/18caf6aac16455780aab75e4afd7f406<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8603-d67701f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8603-d67701f.dmg

**Comment by cozza13** [comment]
tested with a smoke test

**Pull request merged**